includes/polling/functions.inc.php

```diff
@@ -575,6 +584,8 @@ function location_to_latlng($device)
             curl_setopt($curl_init, CURLOPT_CONNECTTIMEOUT, 5);
             $data = json_decode(curl_exec($curl_init), true);
             // Parse the data from the specific Geocode services.
+           d_echo($api_url."\n"); 
```

Comment:
Can you remove these two lines. We'd expose people's API keys here and a lot of users will just upload the output of debug without sanitsing the data.

Reply:
fixed exposed api keys
left print_r($data) for easier troubleshooting
changed geocoding with google without key and set msg "No geocode API key set\n"